README.md

```diff
@@ -0,0 +1,40 @@
+# Blackjack Game
+
+Play Here: https://charlesjbasedow.github.io/blackjack-game/
+
+## Description
+- Player vs Dealer
+- Each receive two cards
+- Player then has the decision to hit(take another card) or stand(keep cards). 
+- Dealer must take cards until it reaches 17 or over 21
+- If either player gets blackjack, they win the hand.
+- If either player goes over 21, they bust(lose)
+- If neither bust, whoever has the highest total wins the hand
+- If both totals are the same, result is a push(tie)
```

Comment:
Suggested edits to this section:
- Player vs. Dealer
- Each receives two cards
- The player then has the decision to hit(take another card) or stand(keep cards).
- Dealer must take cards until it reaches 17 or over 21
- If either player gets blackjack, they win the hand.
- If either player goes over 21, they bust(lose)
- If neither bust, whoever has the highest total wins the hand
- If both totals are the same, the result is a push(tie)